src/index.ts

```diff
@@ -1,8 +1,9 @@
 import { processFile, processFileCRA, processCss } from './transform';
 import { debug } from './preview';
 import { jestPreviewConfigure } from './configure';
+import { enableJestPreview } from './next';
 
-export { jestPreviewConfigure, processCss, processFile, processFileCRA, debug };
+export { jestPreviewConfigure, processCss, processFile, processFileCRA, debug, enableJestPreview };
```

Comment:
I think it can be `configureNextJest`/ `configureJestNext`

Reply:
Done